### PR TITLE
Dev 0.11.0

### PR DESCRIPTION
**Pull request by wForget**
This version has passed the complete functional and non functional tests and meets the release requirements. The test report is shown in the attachment. Welcome to review.（本版本已经经过了完整的功能性和非功能性测试，满足release要求，测试报告如附件所示，欢迎大家审阅。）
[Linkis0.11.0TestReport.docx](https://github.com/WeBankFinTech/Linkis/files/4824181/Linkis0.11.0TestReport.docx)

### CODE REVIEW

**Review by Alexkun — APPROVED** [review]
Thanks.

**Comment by Alexkun** [comment]
The test report is too simple, could you please give a more detailed version, which contains some specific use cases :)

File: core/common/src/main/scala/com/webank/wedatasphere/linkis/common/conf/CommonVars.scala [hunk 1]
```diff
@@ -27,6 +27,12 @@ case class CommonVars[T](key: String, defaultValue: T, value: T, description: St
     if(properties == null || !properties.containsKey(key) || properties.get(key) == null) getValue
     else BDPConfiguration.formatValue(defaultValue, Option(properties.get(key))).get
   }
+
+  def getValueAndRemove(properties: java.util.Map[String, String]): T = {
+    if(properties == null || !properties.containsKey(key) || properties.get(key) == null) getValue
+    else BDPConfiguration.formatValue(defaultValue, Option(properties.remove(key))).get
+  }
+
```
Comment (wushengyeyouya) [inline core/common/src/main/scala/com/webank/wedatasphere/linkis/common/conf/CommonVars.scala:35]: This method should not be added in the core module.

**Review by peacewong — APPROVED** [review]
The newly added es and presto engines are very good features, thank you

**Review by wushengyeyouya — APPROVED** [review]
Lots of great features, thanks very much!

**Pull request merged**